src/main/java/Task/Deadline.java

```diff
@@ -1,3 +1,5 @@
+package Task;
```

Comment:
I noticed that you use inheritance for all the other subclasses and it's clean and easy to read for each individual subclass. Keep up the good work!